reference/5.1/Microsoft.PowerShell.Utility/Write-Verbose.md

```diff
@@ -98,10 +101,15 @@ You can pipe a string that contains the message to `Write-Verbose`.
   more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 - In Windows PowerShell background jobs and remote commands, the `$VerbosePreference` variable in
   the job session and remote session determine whether the verbose message is displayed by default.
-  For more information about the `$VerbosePreference` variable, see [about_Preference_Variables](../Microsoft.PowerShell.Core/About/about_Preference_Variables.md).
+  For more information about the `$VerbosePreference` variable, see
+  [about_Preference_Variables](../Microsoft.PowerShell.Core/About/about_Preference_Variables.md).
 
 ## RELATED LINKS
 
+[about_Output_Streams](/powershell/module/Microsoft.PowerShell.Core/About/about_Output_Streams.md)
```

Comment:
**Site Relative** links shouldn't include the extension.

Reply:
fixed